hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/flow/txn/SkipHandleWorkflow.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.flow.txn;
+
+import com.hedera.hapi.node.base.ResponseCodeEnum;
+import com.hedera.node.app.fees.ExchangeRateManager;
+import com.hedera.node.app.workflows.TransactionInfo;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class SkipHandleWorkflow {
```

Comment:
I'm curious why this class is necessary. It seems like the only extra thing it's doing is putting the `BUSY` response code...

Reply:
Easier to read to know what all happens when older event occurs. Removes long methods.